README.md

```diff
@@ -491,16 +491,17 @@ function accepts a boolean argument that determines whether deprecated versions
 In the following the changes in the respective AST versions, as well as their current support state,
 are listed.
 
-### 80 (experimental)
+### 80 (current)
 
-Available since 1.0.7 (XXX).
+Available since 1.0.7 (2020-07-11).
```

Comment:
Hm ... should this reference the first version where it's available experimentally, or the version that uses the final format? I think it should be the latter.

Reply:
Done, wasn't sure of the warning (Stable since 1.0.7, Supported since 1.0.7), and other entries looked like the first version introducing it)